Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -64,3 +64,205 @@
       - id: FoodCoffeeBeansRaw
     sound:
       path: /Audio/Effects/packetrip.ogg
+
+- type: entity
+  name: avocado
+  parent: FoodProduceBase
+  id: FoodAvocado
+  description: Like delicious buttery grass.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - avocado
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 16
+        reagents:
+        - ReagentId: Nutriment #place holder
+          Quantity: 10
+        - ReagentId: Vitamin
+          Quantity: 6
+  - type: Sprite
+    sprite: _NF/Objects/Specific/Hydroponics/avocado.rsi
+  - type: Produce
+    seedId: avocado
+  - type: SliceableFood
+    count: 2
+    slice: FoodAvocadoSlice
+  - type: Extractable
+    grindableSolutionName: food
+    juiceSolution:
+      reagents:
+      - ReagentId: OilVegetable
+        Quantity: 5
+  - type: Tag
+    tags:
+      - Fruit
+
+- type: entity
+  name: avocado slice
+  parent: ProduceSliceBase
+  id: FoodAvocadoSlice
+  description: Like delicious buttery grass.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - avocado
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
+        - ReagentId: Vitamin
+          Quantity: 3
+  - type: Sprite
+    sprite: _NF/Objects/Specific/Hydroponics/avocado.rsi
+  - type: Extractable
+    grindableSolutionName: food
+    juiceSolution:
+      reagents:
+      - ReagentId: Oil
```

Comment:
Is this supposed to be `Oil` or `OilVegetable`?

Reply:
Yeah, I messed up.  Should be pushed.